server/lib/packets/topic.go

```diff
@@ -0,0 +1,43 @@
+package packets
+
+import (
+	"log"
+
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/models"
+)
+
+func ProcessTopicCreate(from Source, msgid uint32, req *proto.TopicCreateRequest, t Transport) {
```

Comment:
__[gometalinter]__
* [golint] exported function ProcessTopicCreate should have comment or be unexported
* [gotype] undeclared name: proto